src/routes/health/health.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Get, Res } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
+
+@ApiTags('health')
+@Controller({ path: 'health' })
+export class HealthController {
+  @ApiOkResponse({ type: String })
+  @Get()
+  getHealth(@Res() res: Response): void {
+    res.setHeader('Content-Type', 'application/json');
+    res.end('""');
+  }
+}
```

Comment:
@fmrsabino @moisses89 As I said in the description this implementation mimics the behaviour of the Rust client gateway, which returns `""` with a header `Content-Type: application/json`. This feels a bit odd, I don't know this is totally required. If we could return an object instead, the implementation/response would feel more natural in my opinion:

```javascript
  getHealth() {
    return { status: 'OK' };
  }
```

Reply:
I like your suggestion @hectorgomezv , and that shouldn't break anything as in liveness scripts we are only checking the `200` status code

Reply:
Great, that's what I thought, thanks @Uxio0!

Reply:
Yeah for internal related routes we can go about these changes. I don't see any issue with the solution that you propose.